js/views/transactionsVw.js

```diff
@@ -229,7 +229,7 @@ module.exports = baseVw.extend({
       orderID: options.orderID,
       status: options.status,
       serverUrl: this.serverUrl,
-      parentEl: this.$el.find('.js-transactionModalHolder'),
+      parentEl: $('.js-transactionModalHolder'),
```

Comment:
Why move this to index? It should stay in the transaction view, that way if you navigate away from that view it's automatically removed. 

Are there other modals or elements we want the transaction modal to be layered on top of?